### PR TITLE
Added export table w/ data to DAC.

### PR DESCRIPTION
**Pull request by smartin71**
This change supports exporting data with the table, per customer request.

### CODE REVIEW

**Comment by smartin71** [comment]
dac-table.js needs to be updated to agree with Alex's changes before merge.

File: web-client/slycat/web/client/dac_run_chart.py [hunk 1]
```diff
@@ -357,10 +357,11 @@ def read_metadata_table (metadata, arguments, log):
...
     for tdms_type in metadata[0]["tdms_types"]:
         meta_column_names += ['Module ID [' + tdms_type + ']']
-    for tdms_type in metadata[0]["tdms_types"]:
         meta_column_names += ['Module SN [' + tdms_type + ']']
-    for tdms_type in metadata[0]["tdms_types"]:
-        meta_column_names += ['Module ID/SN [' + tdms_type + ']']
+
+    # for tdms_type in metadata[0]["tdms_types"]:
+    #     meta_column_names += ['Module ID/SN [' + tdms_type + ']']
+
```
Comment (Mletter1) [inline web-client/slycat/web/client/dac_run_chart.py:364]: we could probably just delete this right?

Comment (smartin71) [reply web-client/slycat/web/client/dac_run_chart.py:364]: Yes, that would be fine.

Comment (alexsielicki) [reply web-client/slycat/web/client/dac_run_chart.py:364]: I deleted it

**Comment by alexsielicki** [comment]
@smartin71 I resolved the conflict with the changes I made due to SlickGrid upgrade and merged. Please test and make sure it didn't introduce any new bugs. I tried downloading selected rows and entire table and that worked. But checking "Export data with table. Note: this may result in a very large table." resulted in an alert that said "Can't export data -- inconsistent variable units."

**Pull request merged**